pkg/summarizer/summary.go

```diff
@@ -101,18 +101,30 @@ type Fixer func(context.Context, *config.DashboardQueue) error
 // Will use concurrency go routines to update dashboards in parallel.
 // Setting dashboard will limit update to this dashboard.
 // Will write summary proto when confirm is set.
-func Update(ctx context.Context, client gcs.ConditionalClient, mets *Metrics, configPath gcs.Path, concurrency int, tabPathPrefix, summaryPathPrefix string, allowedDashboards []string, confirm bool, features FeatureFlags, freq time.Duration, fixers ...Fixer) error {
+
+type UpdateOptions struct {
+	ConfigPath        gcs.Path
+	Concurrency       int
+	TabPathPrefix     string
+	SummaryPathPrefix string
+	AllowedDashboards []string
+	Confirm           bool
+	Features          FeatureFlags
+	Freq              time.Duration
+}
+
+func Update(ctx context.Context, client gcs.ConditionalClient, mets *Metrics, u UpdateOptions, fixers ...Fixer) error {
```

Comment:
Would be nice to have this func to accept *UpdateOptions instead of the struct itself. This way we will avoid copying the whole struct and instead just pass a pointer. WDYT - @michelle192837 ?

Reply:
+1, I think that fits the existing style elsewhere.